# HG changeset patch
# User Rafael Laboissiere <rafael@laboissiere.net>
# Date 1399277390 -7200
#      Mon May 05 10:09:50 2014 +0200
# Branch stable
# Node ID a9bb634679f8c63004369392cb2c68629942f2f9
# Parent  e3fca7930c1e846acb2c5bbefde4a2cb9596b92b
Fix typo in help string of function figure

diff --git a/scripts/plot/util/figure.m b/scripts/plot/util/figure.m
--- a/scripts/plot/util/figure.m
+++ b/scripts/plot/util/figure.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Command} {} figure
 ## @deftypefnx {Command} {} figure @var{n}
 ## @deftypefnx {Function File} {} figure (@var{n})
 ## @deftypefnx {Function File} {} figure (@dots{}, "@var{property}", @var{value}, @dots{})
-## @deftypefnx {Function File} {@var{h} =} figure (@var{dots})
+## @deftypefnx {Function File} {@var{h} =} figure (@dots{})
 ## Create a new figure window for plotting.
 ##
 ## If no arguments are specified, a new figure with the next available number
 ## is created.
 ##
 ## If called with an integer @var{n}, and no such numbered figure exists, then
 ## a new figure with the specified number is created.  If the figure already
 ## exists then it is made visible and becomes the current figure for plotting.
